mappings/net/minecraft/item/Item.mapping

```diff
@@ -98,24 +186,32 @@ CLASS none/aeo net/minecraft/item/Item
 		ARG 0 world
 		ARG 1 player
 		ARG 2 hand
-	METHOD a (Lnone/aiu;Lnone/aak;Z)Lnone/bcq;
+	METHOD a getHitResult (Lnone/aiu;Lnone/aak;Z)Lnone/bcq;
 		ARG 0 world
 		ARG 1 player
+		ARG 2 hitLiquids
 	METHOD a getItemFromBlock (Lnone/akw;)Lnone/aeo;
 		ARG 0 block
 	METHOD a register (Lnone/akw;Lnone/aeo;)V
 		ARG 0 block
 		ARG 1 itemBlock
 	METHOD a canHarvestBlock (Lnone/asj;)Z
 		ARG 0 state
-	METHOD a (Lnone/dt;)Z
+	METHOD a readTag (Lnone/dt;)Z
+		ARG 0 tag
+	METHOD a getProperty (Lnone/kp;)Lnone/aer;
+		ARG 0 id
+	METHOD a addProperty (Lnone/kp;Lnone/aer;)V
```

Comment:
I went with addProperty, because the method adds it to a list rather than setting a field.